### DISCUSSION TITLE
decreasing HRR for finer grid

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#13675] [Q&A]
Hi all,

I am simulating a lab scale vegetative fire with lagrangian particles. 
Once I refine my grid, the HRR of the fire decreases. 
I also see this in other articles.

What could be the reason of this?

**Comment by Er9y714** [comment]
Is the released mass loss rate of fuel identical? You can check this in _hrr.csv.
Which FDS version are you running?

**Reply by bjmvandongen** [reply]
I am using version 6.9.1. 
No the MLR of the particles also decreases. The fire is propagating less fast which could explain the lower HRR. 
My question now is, why does the fire propagate less fast when a finer grid is used?

**Reply by drjfloyd** [reply]
Change the grid and you have different temperature, heat fluxes, velocities etc. If you are predicting fire spread, you are doing research. Have you looked at particle temperatures, gas phase temperatures, heat fluxes to the particles, etc.? Is you method of particle insertion providing the same amount of fuel mass as you change the grid?

**Comment by mcgratta** [comment]
Create a simple test case and submit it as an issue.
